interpolation/src/lib.rs

```diff
@@ -77,26 +74,26 @@ where
     Mat: Matrix<F>,
 {
     // Slight variation of this approach: https://hackmd.io/@vbuterin/barycentric_evaluation
-    debug_assert_eq!(subgroup.len(), diff_invs.len());
-    debug_assert_eq!(subgroup.len(), coset_evals.height());
+    debug_assert_eq!(coset.len(), diff_invs.len());
+    debug_assert_eq!(coset.len(), coset_evals.height());
 
     let height = coset_evals.height();
     let log_height = log2_strict_usize(height);
 
-    let col_scale: Vec<_> = subgroup
+    let col_scale: Vec<_> = coset
         .par_iter()
         .zip(diff_invs)
         .map(|(&sg, &diff_inv)| diff_inv * sg)
         .collect();
     let sum = coset_evals.columnwise_dot_product(&col_scale);
 
-    let vanishing_polynomial =
-        two_adic_coset_vanishing_polynomial::<EF>(log_height, shift.into(), point);
```

Comment:
This was the only place that `two_adic_coset_vanishing_polynomial` was called, so you can remove its definition and all the associated stuff from `field/src/helpers.rs`, `field-testing/src/lib.rs` and `field/tests/helpers_test.rs`.